plugins/outputs/librato/librato.go

```diff
@@ -17,12 +17,13 @@ import (
 
 // Librato structure for configuration and client
 type Librato struct {
-	APIUser   string `toml:"api_user"`
-	APIToken  string `toml:"api_token"`
-	Debug     bool
-	SourceTag string // Deprecated, keeping for backward-compatibility
-	Timeout   internal.Duration
-	Template  string
+	APIUser   string            `toml:"api_user"`
+	APIToken  string            `toml:"api_token"`
+	Debug     bool              `toml:"debug"`
```

Comment:
not part of your change, but I'm suspicious of this variable now

Reply:
Seems it is needed for print debug message even if write has been successfully done to Librato.